### ISSUE TITLE
Fix autofix for multiline at-rules in indentation

### ISSUE THREAD

**Issue by OriR** [stylelint/stylelint#3274]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Given the config
```json
{
  "extends": ["stylelint-config-strandard"]
}
```
I get these results in my codebase:
![image](https://user-images.githubusercontent.com/2384068/39099239-ca9a4e38-467f-11e8-8a77-5883a6c82381.png)
![image](https://user-images.githubusercontent.com/2384068/39099241-d45740b6-467f-11e8-9051-f6d065049453.png)
![image](https://user-images.githubusercontent.com/2384068/39099242-df2d0598-467f-11e8-8c85-32b0a83507b0.png)
![image](https://user-images.githubusercontent.com/2384068/39099247-f72b2490-467f-11e8-900b-d65045ae61ba.png)
![image](https://user-images.githubusercontent.com/2384068/39099248-fc74a516-467f-11e8-9e97-feecba996f14.png)

As you can see `--fix` is unstable and requires multiple passes to fix rules.
It might be related to `stylelint-config-standard` but I'm not sure, to me it feels like an issue with the linter itself and the order which the rules are placed.

> Which rule, if any, is this issue related to?

Possibly all rules.

> What CSS is needed to reproduce this issue?

```scss
$color-white: white;
$color-accent: blue;

@mixin my-custom-mixin() {
  display: block;
}

.foo {
  @include my-custom-mixin(
      (border: 1px solid transparent, background-color: transparent),
      (border: 1px dashed $color-accent),
      (border: 1px solid $color-accent, background-color: $color-white)
      (border: 1px solid $color-danger, background-color: $color-white),
      (pointer-events: none)
  );
}
```

What you'll see is that for each run the indentation for each mixin argument will change - only after 6 runs it's gonna fix all the errors.

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "extends": ["stylelint-config-strandard"]
}
```

> Which version of stylelint are you using?

`9.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

node_modules/.bin/stylelint "./src/**/*.scss" --fix

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Might be related to scss, but I'm not sure.

> What did you expect to happen?

`--fix` to fix the errors in a single run.

> What actually happened (e.g. what warnings or errors you are getting)?

I had to run the linter multiple times to fix all the errors.

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

With provided example and without `--fix` stylelint reports correctly for each line with incorrect indentation. While with `--fix` option it fixes only the first error, and doesn't fix or show other errors.

Since core team doesn't use non-standard syntaxes like SCSS actively, non-standard syntaxes bugfixes have lower priority. Consider contributing yourself to have fix sooner.

**Comment by OriR** [comment]
@hudochenkov 
I'm able to reproduce this with only css like this:
```css
@supports (
    (perspective: 10px) or
    (-moz-perspective: 10px) or
    (-webkit-perspective: 10px) or
    (-ms-perspective: 10px) or
    (-o-perspective: 10px)) {
  display: block;
}
```
You'll see the same behavior - without `--fix` 5 errors are reported, then with the `--fix` it reports no errors but only fixes the first indented row.

Couldn't reproduce it with a simplified version
```css
.some-class {
    display: block;
    position: relative;
    top: 10px;
    left: 10px;
}
```

So it would appear like an `at-rule` type of thing.
But I don't know what other stylelint rules this affects because right now I only know of the `indentation` that's being broken because of it but it's possible this has a wider impact.

**Comment by jeddy3** [comment]
@OriR Thanks for digging deeper and for reproducing this with standard CSS syntax. I was also able to reproduce it with the following (note the lack of wrapping parens, which is also valid syntax):

```css
@supports
    (perspective: 10px) or
    (-moz-perspective: 10px) or
    (-webkit-perspective: 10px) or
    (-ms-perspective: 10px) or
    (-o-perspective: 10px) {
  display: block;
}
```

Interestingly, in this case it was the first _two_ lines which were automatically fixed.

As such, I'll remove the `non-standard syntax` label. I'm going to add a `help wanted` one, though. Any help investigating this further and fixing it would be greatly appreciated as we're a small team.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and add this.
